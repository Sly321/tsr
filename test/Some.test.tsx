import * as React from "react"
import { shallow, ShallowWrapper } from "enzyme"

describe(`<SomeTest />`, () => {

	describe(`<div>asdf</div>`, () => {
		let tree: ShallowWrapper

		beforeAll(() => {
			tree = shallow(<div className="reality">reality</div>)
		})

		it(`fail test with enzyme matcher className`, () => {
			expect(tree).toHaveClassName("expectation")
		})

		it(`fail test with enzyme matcher text`, () => {
			expect(tree).toHaveText("expectation")
		})

		it(`fail test with enzyme matcher props`, () => {
			expect(tree).toHaveProp("expectation", "expectation")
		})

		it(`create a snapshot`, () => {
			expect(tree).toMatchSnapshot()
		})

	})

})