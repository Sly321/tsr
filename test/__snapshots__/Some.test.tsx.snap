// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SomeTest /> <div>asdf</div> create a snapshot 1`] = `
<div
  className="reality"
>
  reality
</div>
`;
